components/application-operator/pkg/application-controller/reconciler_test.go

```diff
@@ -289,6 +291,7 @@ func TestApplicationReconciler_Reconcile(t *testing.T) {
 		assert.NotNil(t, result)
 		managerClient.AssertExpectations(t)
 		ApplicationReleaseManager.AssertExpectations(t)
+		ApplicationReleaseManager.AssertCalled(t, "UpgradeApplicationRelease", ofType)
```

Comment:
This is not needed AssertExpectations already asserts that everything specified with On and Return was
in fact called as expected.